### DISCUSSION TITLE
Virtual Categories Rule for New Products

### DISCUSSION THREAD

**Discussion by henkvalk** [Smile-SA/elasticsuite#2332] [Ideas]
We love using Virtual Categories. We miss only the option _new products_ inside the rule. 

It would be great to add the rule _New Products_ and as option choose _days old_

Based on the create date of the product ID, you can match the given days old from current date. So it will automatically add new and delete products from the virtual category

**Comment by Bashev** [comment]
This is one way, but for me the proper way is based on "news_from/to_date"

But I know from @romainruaud this "feature" is available in the commerce and have no idea is it possible to be added in the "core" package, or need to be part of additional package.

**Reply by chrisbadley** [reply]
Wrote an extension myself to deal with it based on creation date. Any product with a creation date less than 30 day is marked as new via a "is new" attribute, which in turn is used by elasticsuite to create a virtual category. Once the product is over 30 days old, the attribute is set to no and its removed from the category

**Reply by jg-development** [reply]
@chrisbadley 
Can you provide us with the extension? I would like to test it. It is an interesting approach.
